go/types.go

```diff
@@ -342,3 +342,24 @@ type Precertificate struct {
 func (m *MerkleTreeLeaf) X509Certificate() (*x509.Certificate, error) {
 	return x509.ParseCertificate(m.TimestampedEntry.X509Entry)
 }
+
+type scterror int
+
+//Preallocate errors for performance
+var (
+	INVALID_VERSION   error = scterror(1)
```

Comment:
MixedCaps for constant names too, and golang's pkgs seem to tend to prefix with Err.
e.g. `ErrInvalidVersion`